Modules/overlapped.c

```diff
@@ -1785,8 +1839,13 @@ _overlapped_Overlapped_WSARecvFrom_impl(OverlappedObject *self,
         return NULL;
     }
 
-    wsabuf.len = size;
+    DWORD nread;
```

Comment:
Please revert variable definitions; don't fix if not broken :)
Also, the original form follows the file style: local variables first.

Reply:
Done.